system/Encryption/Handlers/BaseHandler.php

```diff
@@ -61,7 +61,7 @@ protected static function substr($str, $start, $length = null)
      *
      * @param string $key Property name
      *
-     * @return mixed
+     * @return LoggerInterface|null
```

Comment:
`$logger` is null. See #7579
`array|bool|int|string|null` ?

Reply:
Done.